db/views/submissions_by_date_v01.sql

```diff
@@ -13,4 +13,4 @@ GROUP BY
   event_on
 ORDER BY
   application_type,
-  event_on
\ No newline at end of file
+  event_on
```

Comment:
How come there was a view updated here but with no associated migration?

Reply:
The update is just adding a newline to the end of the file

Reply:
Naah the error/warning icon just means no newline